auth/basic-auth/Dockerfile.ppc64le

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.10-alpine3.9 as build
```

Comment:
Is this identical? If so, do we need to maintain a separate Dockerfile?

Reply:
Yes, it is identical. It seems golang is a multiarch image. As for now the only real difference is:

```
16c16
<     go test $(go list ./... | grep -v /vendor/) -cover
---
>     CGO_ENABLED=0 GOOS=linux go test $(go list ./... | grep -v /vendor/) -cover
```

I'm setting the variables CGO_ENABLED and GOOS for go test.

Reply:
We could pass that as an ARG instead and then remove the additional file?

`CGO_ENABLED=0` for `go test` should work for the main x86_64 build. Can you try it?

Reply:
Looks like there's no activity in this PR for a month. I can try this change if it's fine.